src/schema/me/order/index.js

```diff
@@ -0,0 +1,113 @@
+import {
+  GraphQLInt,
+  GraphQLList,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import Artwork from "schema/artwork"
+import EditionSet from "schema/edition_set"
+import Location from "schema/location"
+import Partner from "schema/partner"
+import money from "schema/fields/money"
+import { GravityIDFields } from "schema/object_identification"
+
+export const OrderLineItemType = new GraphQLObjectType({
+  name: "OrderLineItem",
+  fields: () => ({
+    ...GravityIDFields,
+    quantity: {
+      type: GraphQLInt,
+      description: "Quantity of this item",
+    },
+    artwork: {
+      type: Artwork.type,
+      description: "Artwork that is being ordered",
+      resolve: ({ artwork }) => artwork,
```

Comment:
you don't need this resolvers, (I don't think), since they just pluck the same name'd key out of the object (similar to `quantity`).

Reply:
(or does at only work for scalars?)

Reply:
I saw this elsewhere, but you're right! unneeded.